week-4/InClass/D-methods/exercise-5.js

```diff
@@ -17,13 +17,45 @@ let coffeeMachine = {
     },
     insertedAmount: 0,
     insertMoney: function (amount) {
-
+        let insertedAmount = amount;
```

Comment:
Is this the correct way to assign a value to an Object property inside a method i.e. `insertMoney`